lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -261,10 +261,14 @@ spec = do
             , "  --payment PAYMENT        address to send to and amount to send"
             , "                           separated by @, e.g."
             , "                           '<amount>@<address>'"
+            ,"  --withdraw-rewards       Withdraw rewards as change in this"
```

Comment:
```suggestion
            ,"  --withdraw-rewards       Withdraw rewards as a change in this"
```

Reply:
Hmmm. I disagree here, this is not referring to a single change output, but "change" in general.

Reply:
Checked with our fellow native speaker, and he confirms "change is already in a collective context" :+1: